contrib/release.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const child_process = require('child_process');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function prompt(question) {
+  return new Promise(resolve => {
+    readline.question(`${question}: `, response => {
+      readline.close();
+      resolve(response.trim());
+    });
+  });
+}
+
+async function run() {
+  const [, , VERSION] = process.argv;
+
+  if (!VERSION) {
+    throw new Error('error: must specify a valid version');
+  }
+
+  const githash = child_process
+    .execSync('git rev-parse HEAD')
+    .toString()
+    .trim();
+
+  const dir = child_process
+    .execSync(
+      `aws s3 ls realm-clis --recursive | grep ${githash} | sort | tail -n 1 | awk '{print $4}' | cut -f 1-1 -d "/"`
+    )
+    .toString()
+    .trim();
+
+  if (!dir) {
+    throw new Error(`error: failed to find release for git hash ${githash}`);
+  }
+
+  const currentData = JSON.parse(
+    child_process
+      .execSync(`aws s3 cp 's3://realm-clis/versions/cloud-prod/CURRENT' -`)
+      .toString()
+      .trim()
+  );
+
+  const updatedData = JSON.stringify(
+    {
+      version: VERSION,
+      info: {
+        'linux-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/linux-amd64/realm-cli`,
+        },
+        'macos-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/macos-amd64/realm-cli`,
+        },
+        'windows-amd64': {
+          url: `https://s3.amazonaws.com/realm-clis/${dir}/windows-amd64/realm-cli.exe`,
+        },
+      },
+      past_releases: [
```

Comment:
does `past_releases` here want to match `pastReleases` below?

Reply:
in general, all of our public facing API (and DB-level) naming for JSON fields should be snake_cased (we're bad/inconsistent about this... but that was the conclusion we came to a long time ago)

Reply:
i should have been clearer with this comment, I'm fine with either naming style, but I'm guessing both `currentData` and `updatedData` want the same structure, right?

if so, either this field wants to change to `pastReleases` or the line below should update to use `...(currentData.past_releases || []),`

Reply:
ohh yep! good catch!